links.html

```diff
@@ -3,11 +3,12 @@
 <head>
 
 <style>
-section a { 
+#useful-links a { 
```

Comment:
Be as specific as possible with selectors like this.  The selector you used applies to any `a` element inside of any `section` element, but by using `#useful-links` instead of `section` the rule now ONLY applies to the element with id `#useful-links`